lib/plain/endpoints/user.ts

```diff
@@ -7,8 +7,8 @@ export const getForSpace = (http: AxiosInstance, params: GetSpaceParams & { user
   return raw.get<UserProps>(http, `/spaces/${params.spaceId}/users/${params.userId}`)
 }
 
-export const getCurrent = <T = UserProps>(http: AxiosInstance, params?: QueryParams) =>
-  raw.get<T>(http, `/users/me`, { params: params?.query })
+export const getCurrent = (http: AxiosInstance, params?: QueryParams) =>
+  raw.get<UserProps>(http, `/users/me`, { params: params?.query })
```

Comment:
Is there a reason we are reverting this? This allows for us to specify alternative shapes for the response so we can override it in consumers of the client

Reply:
sounds fair. reverting this